apps/ai-image-generator/backend/package.json

```diff
@@ -3,18 +3,29 @@
   "version": "0.1.0",
   "private": true,
   "description": "",
-  "type": "module",
```

Comment:
I really could not get mocha and typescript to work with this option despite lots of trying and floundering.